### ISSUE TITLE
GIS Support AGOL Ownership / Sharing - E. Shackney (Census)

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-geospatial#517]
Elizabeth is working with Mateo on the Census project. She created a Web Mapping Application for the Census Project and needs it shared to the public.

**Comment by jaime-mckeown** [comment]
Shared Web Map and Web Application to the Public:
- Travis Census Census Strategic Map (Web Map / Web App)

**Comment by jaime-mckeown** [comment]
Mateo emailed back that they decided to change administration of the content to planning. I transferred ownership to Derica at planning and cc'd the PAZ GIS SPOC so he's in the loop for administering the content for the Census.

**Comment by jaime-mckeown** [comment]
Mateo emailed back that they decided to change administration of the content to the Planning Department. I transferred ownership to Derica at planning and cc'd the PAZ GIS SPOC so he's in the loop for administering the content for the Census.

**Comment by mateoclarke** [comment]
Thanks Jaime!

**Comment by jaime-mckeown** [comment]
- I should have looked at the data inside the Web Map yesterday as Lizzie had not imported the feature layers she created for the Web Map into her Austin AGOL Account.
- I spoke to Chris D. the PAZ GIS SPOC about this request so he is aware of what's going on and how it will get switched over to them.
- I emailed the group back to ask Lizzie to copy the feature layers over and reconfigure the Web Map so everything can then be switched over to Derica.
- After Lizzie completes the update, Chris will change ownership over to Derica for the Data, Web Map and App and Planning will then take over the entire process.

**Comment by johnclary** [comment]
We discussed today and @jaime-mckeown is going to try to use AGO Assistant to fix the webmap.

**Comment by jaime-mckeown** [comment]
1/15/20
- I looked at fixing the feature layers URLs, but found it to be more complicated as the layers Lizzie uploaded were not the same as the layers in the Web Map. Lizzie and I were emailing so I could get a better understanding before I started changing things. When I went back to the Map to work on it, it had been switched to Derica's account.
- Emailed Derica and she was unaware of the issues with the map. Since she already had the map back under her ownership, we ended up moving all the feature layers under her ownership and the PAZ GIS SPOC Chris D. was going to help her to get it fixed.
- I haven't heard anything back so I'm assuming that they got everything straightened out with the Web Map and feature layers.
